bin/lisky

```diff
@@ -35,6 +38,11 @@ function exit(code) {
 	process.exit(code || 0);
 }
 
+if (!semver.satisfies(process.version, packagejson.engines.node)) {
+	console.error('\x1b[31m', `ERROR: Requires Node.js version ${packagejson.engines.node}, but was started with ${process.version}.`);
```

Comment:
I realise this is a special case because `exit` is called immediately after, but it's always cleaner to include the reset code at the end if you're changing colours this way.

Reply:
Also prefer either `version x.x.x` or `vx.x.x` to be used consistently.